app/assets/javascripts/controllers/paper_edit_controller.js.coffee

```diff
@@ -64,19 +64,21 @@ ETahi.PaperEditController = ETahi.BasePaperController.extend
       @get('visualEditor').startEditing()
 
     toggleEditing: ->
-      if @get('lockedBy') #unlocking
+      if @get('lockedBy') #unlocking -> Allowing others to edit
         @set('body', @get('visualEditor.bodyHtml'))
         @set('lockedBy', null)
         @send('stopEditing')
         @get('model').save().then (paper) =>
           @set('saveState', true)
-      else #locking
+      else #locking -> Editing Paper (locking others out)
         @set('lockedBy', @getCurrentUser())
         @get('model').save().then (paper) =>
           @send('startEditing')
           @set('saveState', false)
 
     savePaper: ->
+      # TODO return if ETahi.environment == 'test'
+      return if ETahi.environment == undefined
```

Comment:
No good option other than this: https://github.com/Tahi-project/tahi/commit/9e1f9973c5faea6f28ac5ac85d7f097007a9b212

Fix the environment check on this line.